tests/mongo-tools/helpers.go

```diff
@@ -49,6 +49,22 @@ func setup(tb testing.TB) (context.Context, *mongo.Client) {
 	return context.Background(), client
 }
 
+// databaseName returns a stable database name for that test.
+func databaseName(tb testing.TB) string {
+	tb.Helper()
```

Comment:
I've seen that we removed the common helpers with the last PR, so I'm assuming that code repetition is ok in that case. If not, let's extract it to some top level helpers file

Reply:
I think we don't need that helper; see the other comment